uniform_isogeny_primes.py

```diff
@@ -75,13 +87,25 @@ def cli_handler(args):  # pylint: disable=redefined-outer-name
     )
     logging.debug("Debugging level for log messages set.")
 
-    eps = get_eps_from_input(args.eps)
+    if args.eps is not None:
+        if args.trace is not None:
+            raise ValueError("--eps and --trace cannot be used at the same time")
+        if args.d < len(args.eps):
+            raise ValueError(f"eps should have length at most d={args.d}, but has length {len(args.eps)} instead")
+        eps = tuple(args.eps + [0] * (args.d - len(args.eps)))
+        do_uniform_eps(args.d, eps, args.trial_division_bound, args.aux_bound)
+        return
 
-    if args.d == int(2):
-        do_uniform_quadratic()
-    else:
-        do_uniform(args.d, eps)
+    if args.trace is not None:
+        if args.trace > 12 * args.d or args.trace < 0:
+            raise ValueError(f"--trace {args.trace} should at most 12*d = {12*args.d} and non-negative.")
+        trace = args.trace if args.trace <= 6 * args.d else 12 * args.d - args.trace
```

Comment:
what is going on here? is this choosing a representative of the trace to be less than 6d?

Reply:
ok